src/libs/ioporder.c

```diff
@@ -191,8 +191,8 @@ void gui_reset (dt_lib_module_t *self)
     dt_dev_pixelpipe_rebuild(darktable.develop);
 
     d->current_mode = DT_IOP_ORDER_V30;
-    if(d->widget)
-      gtk_label_set_text(GTK_LABEL(d->widget), _("v3.0"));
+    if(d->widget) gtk_label_set_text(GTK_LABEL(d->widget), _(dt_iop_order_string(DT_IOP_ORDER_V30)));
```

Comment:
Maybe: `_(dt_iop_order_string(d->current_mode))` to ensure that if the line above is changed we do not forget to update this one. I'm wondering if somehow it is not what has happened here.

Reply:
@TurboGit Pushed new commit with updated and simplification